The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10670786704/job/29575465105) [(plain)](https://github.com/rust-lang/rust/commit/f79a6cd952440b1f8d2bd43401c8ad66464dfcd7/checks/29575465105/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  CACHE_DOMAIN: ci-caches.rust-lang.org
  PR_CI_JOB: 1
  IMAGE: mingw-check-tidy
##[endgroup]
fatal: unknown commit 53fd98ca776cb875bc9e5514f56b52eb74f9e7a9
All commits in `HEAD` are present in `master`
src/ci/scripts/verify-stable-version-number.sh
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#13 2.912 Building wheels for collected packages: reuse
#13 2.914   Building wheel for reuse (pyproject.toml): started
#13 3.161   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.162   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#13 3.162   Stored in directory: /tmp/pip-ephem-wheel-cache-43dqt5pi/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#13 3.165 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#13 3.565 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#13 3.565 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 4.083 Collecting virtualenv
#13 4.083 Collecting virtualenv
#13 4.131   Downloading virtualenv-20.26.3-py3-none-any.whl (5.7 MB)
#13 4.205      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.7/5.7 MB 79.6 MB/s eta 0:00:00
#13 4.262 Collecting filelock<4,>=3.12.2
#13 4.265   Downloading filelock-3.15.4-py3-none-any.whl (16 kB)
#13 4.295 Collecting platformdirs<5,>=3.9.1
#13 4.298   Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
#13 4.315 Collecting distlib<1,>=0.3.7
#13 4.327      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 83.3 MB/s eta 0:00:00
#13 4.327      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 83.3 MB/s eta 0:00:00
#13 4.408 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 4.599 Successfully installed distlib-0.3.8 filelock-3.15.4 platformdirs-4.2.2 virtualenv-20.26.3
#13 DONE 4.7s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      245696 kB
DirectMap2M:     8142848 kB
DirectMap1G:    10485760 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
thread 'main' panicked at src/lib.rs:1507:38:
thread 'main' panicked at src/lib.rs:1507:38:
called `Result::unwrap()` on an `Err` value: ParseIntError { kind: Empty }
Build completed unsuccessfully in 0:00:00
  local time: Mon Sep  2 17:03:10 UTC 2024
  network time: Mon, 02 Sep 2024 17:03:10 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/src/bootstrap/src/lib.rs
+++ b/src/bootstrap/src/lib.rs
@@ -1501,6 +1501,7 @@ Executed at: {executed_at}"#,
                     "refs/remotes/origin/{}..HEAD",
                     self.config.stage0_metadata.config.nightly_branch
                 ))
+                .run_always()
                 .run_capture(self)
                 .stdout()
         });

